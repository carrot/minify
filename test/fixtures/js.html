<script>
  (function () {
    var aLongVariableName = 'wow'
    return aLongVariableName
  })()
</script>
