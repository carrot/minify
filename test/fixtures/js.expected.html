<script>!function(){var n="wow";return n}();</script>
